Automattic/jetpack
# Search: move search dashboard to package

#### Changes proposed in this Pull Request:

- Moved Search dashboard to search the package
  - Created a lighter state store utilising `@wordpress/data`
  - Copied and trimmed some dependencies from `plugins/jetpack/_inc/client`
  - New Webpack config `packages/search/build/webpack.search-dashboard.config.js`
  - Added success/error notices
- Added `fetchSearchPlanInfo`, `fetchSearchSettings`,`updateSearchSettings` in js-package `api` and bumped dependent versions
- Removed replaced search admin code in Jetpack the plugin

TODO
- [ ] Wait for API PR to be merged
- [ ] Resolve duplicate packages
- [ ] Make sure Search the package builds with Jetpack the plugin

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Change current directory to `vendor/automattic/jetpack-search` and run `pnpm build-dashboard`
- Navigate to `/wp-admin/admin.php?page=jetpack-search`
- Ensure the page still works well and looks well
